src/cplus_plugin/models/helpers.py

```diff
@@ -141,10 +141,18 @@ def create_ncs_pathway(source_dict) -> typing.Union[NcsPathway, None]:
     :rtype: NcsPathway
     """
     ncs = create_layer_component(source_dict, NcsPathway)
+
+    # We are checking because of the various iterations of the attributes
+    # in the NcsPathway class where some of these attributes might
+    # be missing.
     carbon_paths_attr = "carbon_paths"
```

Comment:
@gkahiu now that we have time for code structure improvements, lets make minor/small improvements in the codebase. We need to define all the strings literal in the `constants.py` file eg. for here the `carbon_paths` string should have its own constant variable.

Reply:
Done.